ingestion-beam/src/test/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRowTest.java

```diff
@@ -427,7 +460,7 @@ public void testDoublyNestedList() throws Exception {
         + "}\n").getBytes(StandardCharsets.UTF_8));
     List<Field> bqFields = ImmutableList.of(Field.newBuilder("payload", LegacySQLTypeName.RECORD, //
         Field.newBuilder("list", LegacySQLTypeName.RECORD, //
-            Field.of("list", LegacySQLTypeName.INTEGER)) //
+            Field.newBuilder("list", LegacySQLTypeName.INTEGER).setMode(Mode.REPEATED).build()) //
```

Comment:
This field should have been defined as REPEATED all along in this test, but the new code changes introduced in this PR caused the inner lists to get incorrectly interpreted as int fields and sent to additional_properties. Changing the mode here causes the test to pass again.